cypress/integration/distroSettings/host_section.ts

```diff
@@ -57,4 +57,55 @@ describe("host section", () => {
       cy.validateToast("success");
     });
   });
+
+  describe("using User Data bootstrap method", () => {
+    beforeEach(() => {
+      cy.visit("/distro/ubuntu1604-parent/settings/host");
```

Comment:
Agree this would be helpful, but Cypress totally chokes when working with any value from `constants/routes`. Would prefer to resolve this separately!

```
Error: Webpack Compilation Error
./src/utils/string/githubPRLinkify.tsx 10:158
Module parse failed: Unexpected token (10:158)
File was processed with these loaders:
 * ../../Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/packages/server/node_modules/ts-loader/index.js
You may need an additional loader to handle the result of these loaders.
| var styles_1 = require("components/styles");
| var githubPRLinkify = function (unlinkified) {
>     return (0, react_string_replace_1.default)(unlinkified, /(https:\/\/github.com\/[a-zA-Z0-9-]+\/[a-zA-Z0-9-]+\/pull\/\d+)/g, function (match, i) { return (<styles_1.StyledLink key={"".concat(match).concat(i)} href={match}>
|         {match}
|       </styles_1.StyledLink>); });
 @ ./src/utils/string/index.ts 9:24-52
 @ ./src/utils/index.ts
 @ ./src/constants/routes.ts
 @ ./cypress/integration/distroSettings/host_section.ts
    at Watching.handle [as handler] (/Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/packages/server/node_modules/@cypress/webpack-preprocessor/dist/index.js:212:23)
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Watching.js:99:9
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Watching._done (/Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Watching.js:98:28)
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Watching.js:73:19
    at Compiler.emitRecords (/Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:499:39)
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Watching.js:54:20
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:485:14
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:482:27
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/neo-async/async.js:2818:7
    at done (/Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/neo-async/async.js:3522:9)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:464:33
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/graceful-fs.js:143:16
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/graceful-fs.js:143:16
    at /Users/sophstad/Library/Caches/Cypress/12.17.1/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/graceful-fs.js:61:14
    at FSReqCallback.oncomplete (node:fs:196:23)
```